README.md

```diff
@@ -146,7 +157,8 @@ modelBuilder
     .HasData([
         new()
         {
-            Id = 1, ExternalId = new("d578489e45e04ff89ef65b529ed5d95c"), 
+            Id = 1, 
+            ExternalId = new("d578489e45e04ff89ef65b529ed5d95c"), 
```

Comment:
Comment that this ID is an arbitrary guid. Most IDEs can generate those with `nguid` snippet.